File: reporte/src/main/java/plataforma/reporte/model/Catedra.java
package plataforma.reporte.model;

import java.util.Arrays;
import java.util.List;

public class Catedra {
    public int id;
    public Turno turno;
    public Profesor profesor;
    public Materia materia;
    public Cuatrimestre cuatrimestre;
    public Dia dia;
    public boolean es_final;
    public Object fecha_final;

    @Override
    public String toString() {
        return "Catedra{" +
                "id=" + id +
                ", turno=" + turno +
                ", profesor=" + profesor +
                ", materia=" + materia +
                ", cuatrimestre=" + cuatrimestre +
                ", dia=" + dia +
                ", es_final=" + es_final +
                ", fecha_final=" + fecha_final +
                '}';
    }
}

File: estudiante/Estudiante-Service/Estudiante-Service/App_Code/Models/Catedra.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;

public class Catedra
{
    public int idcatedra { get; set; }
    public string materia { get; set; }
    public string profesor { get; set; }
    public string turno { get; set; }
    public string dia { get; set; }
    public DateTime fecha { get; set; }
    public string cuatrimestre { get; set; }
    public string inscripto { get; set; }
    public bool es_final { get; set; }
    [IgnoreDataMember]
    public int idInscripcion { get; set; }
    [IgnoreDataMember]
    public int idmateria { get; set; }
    [IgnoreDataMember]
    public int idEstudiante { get; set; }
}